pkg/ddevapp/drupal.go

```diff
@@ -341,10 +342,18 @@ func GetDrupalVersion(app *DdevApp) (string, error) {
 	return v, err
 }
 
-// isDrupalApp returns true if the app is drupal
+// isDrupalApp returns true if the app is drupal (drupal8+)
 func isDrupalApp(app *DdevApp) bool {
-	v, err := GetDrupalVersion(app)
-	if err == nil && v != "" {
+	vStr, err := GetDrupalVersion(app)
+	if err != nil {
+		return false
+	}
+	v, err := strconv.Atoi(vStr)
```

Comment:
I think we can return int from `GetDrupalVersion()` instead.

This will require more changes elsewhere, but should be better in the long run, especially for comparing numbers.

Reply:
Oh, forget it, I see checks for `drupalVersion == ""`, so empty string should be here anyway.